play-ws-standalone/src/main/scala/play/api/libs/ws/StandaloneWSRequest.scala

```diff
@@ -138,7 +146,17 @@ trait StandaloneWSRequest {
     val newHeaders = headers.toList.flatMap { param =>
       param._2.map(p => param._1 -> p)
     } ++ hdrs
-    withHttpHeaders(newHeaders: _*)
+
+    // Content-Type is single valued:
+    // https://tools.ietf.org/html/rfc7231#section-3.1.1.5
+    //
+    // So we should undo the append made above.
```

Comment:
We are "undoing the append", discarding the old value and using just the new one. That way we keep `Content-Type` with a single value. The problem with append for `Content-Type` is that, later, when building the request, we use the [first value for the header](https://github.com/playframework/play-ws/blob/master/play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StandaloneAhcWSRequest.scala#L50).

In this case, I think we should update the scaladocs.